packages/modules/desktop-viewer-react/README.md

```diff
@@ -1,9 +1,104 @@
 # iTwin Viewer for Desktop
 
-Consider this package as unstable until 1.0 release.
```

Comment:
Is this moving to 1.0?

Reply:
it already is. Just forgot to remove this line

Reply:
ah okay